appmap/django.py

```diff
@@ -288,7 +288,6 @@ def inject_middleware():
 def load_middleware(*args, **kwargs):
     """Patched wrapper to inject AppMap middleware first"""
     inject_middleware()
-    BaseHandler.load_middleware = original_load_middleware
```

Comment:
@divide resetting to the original `load_middleware` function here causes problems if the test sets the middleware (as happens here https://github.com/mirumee/saleor/blob/master/saleor/core/tests/test_middleware.py#L16).

Did you do this just for the sake of efficiency, or was there some other reason for it?